web-app/CKS.Web.Test/IntegrationTests/Infrastructure/CKSWebApplicationFactory.cs

```diff
@@ -19,17 +19,30 @@ public class CKSWebApplicationFactory : WebApplicationFactory<Startup>
 	{
 		private readonly ITestOutputHelper _output;
 		private readonly IList<Action<IServiceCollection>> serviceCollectionsActions = new List<Action<IServiceCollection>>();
+		private string webRootPath = null;
 
 		public CKSWebApplicationFactory(ITestOutputHelper output)
 		{
 			_output = output;
 			_output.WriteLine("CKSWebApplicationFactory.constructor");
 		}
 
+		protected override IWebHostBuilder CreateWebHostBuilder() => base.CreateWebHostBuilder();
+
 		protected override void ConfigureWebHost(IWebHostBuilder builder)
 		{
 			_output.WriteLine("CKSWebApplicationFactory.ConfigureWebHost");
 
+			builder.UseEnvironment("Testing");
```

Comment:
The reverse comment from before - if we keep this here then we don't need line 30. Or we change line 30 to call `UseEnvironment` in which case we remove this line.

Reply:
interestingly if we use line 30 the environment is never actually set to testing.